### PR TITLE
Prettier

### PR DESCRIPTION
**Pull request by spicymaya**
All descriptions in commit messages. Please take a look.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,26 +4,37 @@
...
   "devDependencies": {
     "@babel/core": "^7.2.2",
     "@babel/plugin-proposal-class-properties": "^7.3.0",
+    "@babel/polyfill": "^7.2.5",
     "@babel/preset-env": "^7.3.1",
     "@babel/preset-react": "^7.0.0",
+    "@babel/register": "^7.0.0",
     "awesome-typescript-loader": "^5.2.1",
     "babel-loader": "^8.0.5",
+    "babel-register": "^6.26.0",
```
Comment (dialupdev) [inline package.json]: Do you need this one?

File: package.json [hunk 2]
```diff
@@ -4,26 +4,37 @@
...
     "css-loader": "^2.1.0",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.9.1",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
+    "ignore-styles": "^5.0.1",
```
Comment (dialupdev) [inline package.json]: What does this do?

File: src/js/components/SingleFood.jsx [hunk 3]
```diff
@@ -1,19 +1,7 @@
...
-  Container,
-  Row,
-  Col,
-  Card,
-  CardImg,
-  CardText,
-  CardBody,
-  CardTitle,
-  CardSubtitle,
-  Badge
```
Comment (dialupdev) [inline src/js/components/SingleFood.jsx:12]: 🥭

**Review by dialupdev — APPROVED** [review]
Looks super nice!!! Really good work. https://bukk.it/twinpeaks-bread.gif

File: package.json (hunk 1 above)
Comment (spicymaya) [reply package.json]: Need this for jasmine tests

File: package.json (hunk 2 above)
Comment (spicymaya) [reply package.json]: Need this for Jasmine tests

**Pull request merged**